package.json

```diff
@@ -140,16 +140,16 @@
   },
   "lint-staged": {
     "{src,src/**,src/**/**,assets/,assets/**}/*.{js,jsx,scss}": [
-	  "prettier --write",
-	  "git add"
+      "prettier --write",
+      "git add"
     ],
     "assets/scss/**/*.scss": [
-	  "stylelint --syntax scss --fix",
-	  "git add"
+      "stylelint --syntax scss --fix",
+      "git add"
     ],
     "**/*.{js,jsx}": [
-	  "eslint",
-	  "git add"
+      "eslint",
+      "git add"
     ]
   }
-}
\ No newline at end of file
+}
```

Comment:
Considering the problems that Dr. Mike has been having with mwp packages, you should probably have him look at the package.json and swarm_constants files to confirm the changes are necessary.

Reply:
Well @browne0 guided me through this, it's basically just creating a stub for the constants file, if I didn't do that it would complain when trying to import the module into `Dropdown.jsx`

Reply:
The problems were more with code changes in mwp, these should be fine :)